app/assets/stylesheets/events.css.scss

```diff
@@ -213,6 +213,17 @@ article.type-system-geometric {
   }
 }
 
+#thankYous {
+  #thankYou {
+    text-align: center;
+    font-size: 6.5em;
+  }
+  #thankYouAccry {
```

Comment:
Avoid using id selectors<br>Selector `thankYouAccry` should be written in lowercase with hyphens